src/main/java/org/isf/sms/providers/skebby/SkebbyGatewayService.java

```diff
@@ -44,16 +44,16 @@
 @PropertySource("classpath:sms.properties")
 public class SkebbyGatewayService implements SmsSenderInterface {
 
-	private static final String SERVICE_NAME = "skebby-gateway-service";
-	private static final String RESPONSE_SUCCESS = "OK";
-	private static final String KEY_PASSWORD = "skebby-gateway-service.password";
-	private static final String KEY_USERNAME = "skebby-gateway-service.username";
-	private static final String KEY_MESSAGE_TYPE = "skebby-gateway-service.message-type";
+	public static final String SERVICE_NAME = "skebby-gateway-service";
+	public static final String RESPONSE_SUCCESS = "OK";
+	public static final String KEY_PASSWORD = "skebby-gateway-service.password";
+	public static final String KEY_USERNAME = "skebby-gateway-service.username";
+	public static final String KEY_MESSAGE_TYPE = "skebby-gateway-service.message-type";
```

Comment:
are we sure these fields should be public?

Reply:
I made them public so I could use them in the tests without duplicating the strings.

Reply:
yes I imagined that, but since are very sensitive fields shouldn't we use getters?

Reply:
If we used getters they would have to be `public` too.   So instead I made them protected and moved the test into the same package structure as the `SkebbyGatewayService.java` file.

Reply:
No; I think that the tests really should follow the main structure/layout so access to protected things are easier.   I would eventually like to make all the tests look more like this.

Reply:
shouldn't we do it in one shot later on instead to apply the new pattern only to this package?

Reply:
It is already done in other packages, for example, generaldata and utils.